vscode/src/config.ts

```diff
@@ -26,3 +26,18 @@ export async function setTarget(target: TargetProfile) {
     vscode.ConfigurationTarget.Global,
   );
 }
+
+export function getEnableFormating(): boolean {
+  return vscode.workspace
+    .getConfiguration("Q#")
+    .get<boolean>("enableFormatting", true);
+}
+
+export async function setEnableFormating(isEnabled: boolean) {
```

Comment:
Does this need to be implemented for any reason? Does anything call this?

Reply:
Hmm, I thought I had this being set somewhere programmatically, but looking now, I guess not. Removed.